### PR TITLE
Reorder Scripts (Debug PR #2938)

### PR DESCRIPTION
**Pull request by sanssecours**

### CODE REVIEW

**Comment by markus2330** [comment]
Thank you for working on it! You can also directly push to the "scripts" branch. Or should I remove the branch?

**Comment by sanssecours** [comment]
> You can also directly push to the "scripts" branch. Or should I remove the branch?

For now it would be great, if we can keep the `scripts` branch, since I am still in the process of cherry-picking commits from there.

File: doc/tutorials/elektra-merge-integration.md [hunk 1]
```diff
@@ -1,104 +0,0 @@
-# How-To: Integrate elektra-merge Into a Debian Package
```
Comment (markus2330) [inline doc/tutorials/elektra-merge-integration.md]: @Chemin1 we will remove this for now, could you please write a similar tutorial?

**Comment by markus2330** [comment]
@sanssecours looks very good, the build passes now.

File: doc/tutorials/elektra-merge-integration.md (hunk 1 above)
Comment (Chemin1) [reply doc/tutorials/elektra-merge-integration.md]: @markus2330  Yes. I opened a new issue and added it to the project so that the task is not forgotten.

Comment (markus2330) [reply doc/tutorials/elektra-merge-integration.md]: Thank you! I hope you had nice holidays!

**Pull request merged**